inter-server-sync.spec

```diff
@@ -0,0 +1,109 @@
+#
+# spec file for package uyuni inter server sync
+#
+# Copyright (c) 2021 SUSE LLC
```

Comment:
Let's use the opportunity to bump the copyright?

Reply:
sure